heron/common/src/java/org/apache/heron/common/utils/metrics/BoltMetrics.java

```diff
@@ -127,6 +130,7 @@ public void failedTuple(String streamId, String sourceComponent, long latency) {
   public void executeTuple(String streamId, String sourceComponent, long latency) {
     executeCount.incr();
     executeLatency.update(latency);
+    executeTimeNs.incrBy(latency);
```

Comment:
is this equation correct ? executeLatency x executeCount = executeTimeNs

Reply:
correct in most cases. One exception is that count.incr() and latency().update() are not atomic, so in theory they might be separated into two intervals. This

The more straightforward motivation is more like the FullBoltMetrics class has the metric but the others dont. This PR makes them more consistent.